src/klein/test/test_request_compat.py

```diff
@@ -143,19 +147,19 @@ def test_bodyAsBytes(self, data: bytes) -> None:
         """
         legacyRequest = self.legacyRequest(body=data)
         request = HTTPRequestWrappingIRequest(request=legacyRequest)
-        body = self.successResultOf(request.bodyAsBytes())
+        body = self.successResultOf(ensureDeferred(request.bodyAsBytes()))
 
         self.assertEqual(body, data)
 
-    def test_bodyAsBytesCached(self) -> None:
+    async def test_bodyAsBytesCached(self) -> None:
```

Comment:
@wsanchez async tests are not tested yet https://github.com/twisted/twisted/pull/1399

Reply:
Whoops, I knew that and forgot the decorator, thanks